can_tools/scrapers/official/ND/common.py

```diff
@@ -0,0 +1,60 @@
+# ND County Populations
+# Retrieved from ND COVID Case Count Dashboard
+# https://www.health.nd.gov/diseases-conditions/coronavirus/north-dakota-coronavirus-cases
+# Based on 2018 counts
+
+nd_populations = {
```

Comment:
Are we worried about population fluctuation?

Reply:
I don't think so. For me this is an acceptable way to move from percent to counts